controllers/API/comment.js

```diff
@@ -38,21 +39,21 @@ router.post("/API/forum/comment", function (req, res) {
 });
 
 router.delete("/API/forum/comment", function (req, res) {
-	Comment.findOne({"_id": req.query.id}, function (err, comment) {
+	Comment.findOne({ "_id": req.query.id }, function (err, comment) {
```

Comment:
`"object-curly-spacing": ["error", "always"]` this needs to be added to `.eslintrc.js`